flexibleengine/resource_flexibleengine_dcs_instance_v1.go

```diff
@@ -196,7 +197,45 @@ func getInstanceBackupPolicy(d *schema.ResourceData) *instances.InstanceBackupPo
 	return instanceBackupPolicy
 }
 
+func getDcsProductId(d *schema.ResourceData, meta interface{}) error {
```

Comment:
no need to get the client twice, we can define the function as: ```func getDcsProductId(client *golangsdk.ServiceClient, instanceType string) (string, error) { ... }```

Reply:
ok

Reply:
> no need to get the client twice, we can define the function as: `func getDcsProductId(client *golangsdk.ServiceClient, instanceType string) (string, error) { ... }`

ok